avd-containers/dev/.devcontainer/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+AVD_DEV_REQ_FILE="${HOME}/.ansible/collections/ansible_collections/arista/avd/requirements-dev.txt"
+USERNAME=$(whoami)
+
+# install ansible if not yet installed and avd collection is locally mounted
+ansible --version &> /dev/null ||  if [ -f $AVD_DEV_REQ_FILE ]; then
```

Comment:
I don't think we need to do all this, since you install from requirements-dev.txt anyway. Instead just install the latest version of ansible-core, and then install the collections and python requirements. That may downgrade ansible-galaxy again, but so be it.

Reply:
Well, we can install the correct galaxy version from the start and it takes nothing. This small piece of bash code is well tested and not causing any troubles.